lib/spack/spack/build_systems/r.py

```diff
@@ -33,6 +41,34 @@ class RPackage(PackageBase):
 
     extends('r')
 
+    @property
+    def homepage(self):
+        if self.cran:
+            return 'https://cloud.r-project.org/package=' + self.cran
+        elif self.bioc:
+            return 'https://bioconductor.org/packages/' + self.bioc
+
+    @property
+    def url(self):
+        if self.cran:
+            return (
+                'https://cloud.r-project.org/src/contrib/'
+                + self.cran + '_1.2.3.tar.gz'
```

Comment:
Does `self.versions` contain a dict of versions we can access? It doesn't really matter if we use a fake version, but `spack url summary` uses these numbers to compute our url parsing accuracy.

Reply:
I am not following what you mean.

Reply:
Do you mean something like this:
```python
    @property
    def url(self):
        if self.cran:
            for version in self.versions:
                return (
                    'https://cloud.r-project.org/src/contrib/'
                    + self.cran + '_' + str(version) + '.tar.gz'
                )
```

Reply:
Something like:
```python
    @property
    def url(self):
        if self.cran:
            return (
                'https://cloud.r-project.org/src/contrib/'
                + self.cran + '_' + str(self.versions[0]) + '.tar.gz'
            )
```
Just so it isn't an arbitrary fake version, but an actual version from the package.

Reply:
Okay; I understand. I thought you wanted the whole dict of versions, which is why I was confused.